go/adbc/driver/bigquery/statement.go

```diff
@@ -495,6 +513,127 @@ func arrowDataTypeToTypeKind(field arrow.Field, value arrow.Array) (bigquery.Sta
 	}
 }
 
+func arrowFieldToBigQueryField(arrowField arrow.Field) (bigquery.FieldSchema, error) {
```

Comment:
This is imperfect but it's working for several use cases right now. I have a lot of research I'd want to do on the bigquery SDK to remark this. There are multiple possible possibles and I think this one is an older one.